packages/design-system/src/components/Icons/PlusCircleIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { t } from '../i18n';
+import SvgIcon, { IconCommonProps } from './SvgIcon';
+import classNames from 'classnames';
+
+const defaultProps = {
+  className: '',
+  viewBox: '0 0 20 20',
+};
+
+function PlusCircleIcon(props: IconCommonProps): React.ReactElement {
+  const iconCssClasses = classNames('ds-c-icon--plus-circle', props.className);
+
+  return (
+    <SvgIcon title={t('icons.add')} {...defaultProps} {...props} className={iconCssClasses}>
+      <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+        <g transform="translate(-47.000000, -360.000000)">
+          <g transform="translate(49.000000, 362.000000)">
+            <circle
+              className="PlusCircleIcon-Circle"
+              stroke="#0071BB"
+              strokeWidth="2"
+              fill="#FFFFFF"
+              cx="8"
+              cy="8"
+              r="9"
+            ></circle>
+            <g
+              className="PlusCircleIcon-Group"
+              transform="translate(2.823529, 2.823529)"
+              fill="#0071BB"
```

Comment:
Please update the value of `fill` to be `currentColor`. This will allow the icon's color to be changed in CSS by setting the color value. If you need the icon to be blue for the accordion, you can set the color value in the accordion's styles.